ldelf/ldelf.mk

```diff
@@ -29,6 +29,10 @@ cppflags$(sm)	+= -include $(conf-file)
 cppflags$(sm)	+= -DTRACE_LEVEL=$(CFG_TEE_CORE_LOG_LEVEL)
 cppflags$(sm)	+= -D__LDELF__
 
+ifeq ($(CFG_TA_SANITIZE_UNDEFINED),y)
```

Comment:
Please use `CFG_CORE_SANITIZE_UNDEFINED` for LDELF.

Reply:
Done